pyqmc/orbitals.py

```diff
@@ -55,6 +55,8 @@ def choose_evaluator_from_pyscf(
             determinants = pyqmc.determinant_tools.pbc_determinants_from_casci(
                 mc, mc.orbitals
             )
+            if hasattr(mc, "mo_coeff"):
```

Comment:
I don't think it's a good idea to have side effects; this changes the mf object.

Reply:
i changed it so that it doesn't change the mf object.